ion/symboltoken.go

```diff
@@ -103,6 +112,15 @@ func symbolIdentifier(symbolText string) (int64, bool) {
 	return SymbolIDUnknown, false
 }
 
+// NewSimpleSymbolToken returns a Symbol Token with the given text value and undefined SID and Source.
+func NewSimpleSymbolToken(text string) SymbolToken {
```

Comment:
This is subjective, but to me the phrase `Simple` doesn't communicate what this function will do. How about `NewSymbolTokenFromString`?

Reply:
Done.